presto-main/src/main/java/com/facebook/presto/statusdetector/NodeStatusDetectorConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.statusdetector;
+
+import com.facebook.airlift.configuration.Config;
+
+public class NodeStatusDetectorConfig
+{
+    private boolean enabled;
+
+    public boolean isEnabled()
```

Comment:
Are we using this anywhere. ?
Shouldn't the filtering logic in `DiscoveryNodeManager.java` respect this flag?

Reply:
made changes, please have a look at it again.